api/http/dataplane/data-plane-api.yaml

```
@@ -145,26 +145,38 @@ components:
     ConfigurationStatus:
       type: object
       properties:
```

Should we make some of these fields required (so they aren't pointers in our generated code)?
```suggestion
    ConfigurationStatus:
      type: object
      required:
        - instanceId
        - correlationId
        - events
      properties:
```